Cargo.toml

```diff
@@ -95,7 +95,7 @@ tlvc = { git = "https://github.com/oxidecomputer/tlvc" }
 tlvc-text = {git = "https://github.com/oxidecomputer/tlvc"}
 vsc7448-info = { git = "https://github.com/oxidecomputer/vsc7448.git" }
 vsc7448-types = { git = "https://github.com/oxidecomputer/vsc7448.git" }
-ipcc-data = { git = "https://github.com/oxidecomputer/ipcc-data" }
+ipcc-data = { git = "https://github.com/oxidecomputer/ipcc-rs" }
```

Comment:
Do we want to consider pinning this to a specific commit, especially in a shared repo?

Reply:
It's implicitly pinned, because we commit `Cargo.lock` (unlike `sprockets`), so I think it's fine!